apps/shared/src/components/status-header/StatusHeader.tsx

```
@@ -0,0 +1,248 @@
+ * StatusHeader — State badge and elapsed-time display for a pipeline.
+ * StatusHeaderInfo / StatusElapsed — the header's composable pieces, exported
+ * so hosts that render inside a stock Card's header/headerActions slots reuse
+ * the SAME markup instead of nesting a second header row.
+ * StatusActions — Run/Stop button (stock Button, per the Card-header rule
+ * that header buttons are plain `<Button small>`).
+ */
+
+import React from 'react';
+import type { CSSProperties, ReactNode } from 'react';
+import PadlockIcon from '../../assets/icons/PadlockIcon';
+import type { ITaskStatus } from 'shell';
```

ESLint

[error] 'shell' import is restricted from being used. Renamed: import from 'shell'.

(no-restricted-imports)